ledger_device_sdk/src/nbgl.rs

```diff
@@ -177,32 +177,30 @@ const INFO_FIELDS: [*const c_char; 2] = [
 /// Used to display the home screen of the application, with an optional glyph,
 /// information fields, and settings switches.  
 pub struct NbglHomeAndSettings<'a> {
-    app_name: *const c_char,
-    icon: *const nbgl_icon_details_t,
```

Comment:
Not sure replacing the icon field with a local variable in the `show` function will always work (both in `debug` and `release` compilation). I had many issues when passing local variables to the NBGL API where it made the app crash or had display issues. 

Did you verify the behavior of the app in functional tests (preferrably with the `release`  mode binary) with this change ?